src/azul/__init__.py

```diff
@@ -1520,6 +1524,64 @@ def vpc_cidr(self) -> str:
     def vpn_subnet(self) -> str:
         return self.environ['azul_vpn_subnet']
 
+    @property
+    def content_security_policy(self) -> str:
```

Comment:
This makes zero sense here. It should be in `browser.tf.json.template.py`, just not at the beginning, but with the other utility functions there.